pkg/uefi/meregion.go

```diff
@@ -95,15 +95,20 @@ func (e MEPartitionEntry) Type() string {
 
 // FindMEDescriptor searches for an Intel ME FTP signature
 func FindMEDescriptor(buf []byte) (int, error) {
-	if bytes.Equal(buf[16:16+len(MEFTPSignature)], MEFTPSignature) {
+	if bytes.Equal(buf[16:16+len(MEFPTSignature)], MEFPTSignature) {
 		// 16 + 4 since the descriptor starts after the signature
-		return 16 + len(MEFTPSignature), nil
+		return 16 + len(MEFPTSignature), nil
 	}
-	if bytes.Equal(buf[:len(MEFTPSignature)], MEFTPSignature) {
+	if bytes.Equal(buf[:len(MEFPTSignature)], MEFPTSignature) {
 		// + 4 since the descriptor starts after the signature
-		return len(MEFTPSignature), nil
+		return len(MEFPTSignature), nil
 	}
-	return -1, fmt.Errorf("ME Flash Partition Table signature %#02x not found: first 20 bytes are:\n%s", MEFTPSignature, hex.Dump(buf[:20]))
+	fptOffset := bytes.Index(buf, MEFPTSignature)
+	if fptOffset > 0 {
+		fmt.Printf("FPT Signature @ 0x%x\n", fptOffset)
```

Comment:
this Printf in a package is a Nono

Reply:
oops leftover from debugging - will take that out